docs/ci_runner_setup/linux/README.md

```diff
@@ -0,0 +1,78 @@
+## Linux CI runner setup
+
+This directory contains documentation and scripts about setting up a Linux CI Runner for [`ROCm`](https://github.com/ROCm) organization and used by [`TheRock`](https://github.com/ROCm/TheRock) repository.
+
+Note: you must have sufficient permissions to access [ROCm runner page](https://github.com/organizations/ROCm/settings/actions/runners)
+
+### Setup
+
+For brand new machines that do not that ROCm or Docker installed, please follow these steps. Otherwise, please skip to step 3.
```

Comment:
Grammar/typo errors in this line, but also see my other comments about ROCm installs.